cas-server-support-osf/src/main/java/io/cos/cas/web/flow/OpenScienceFrameworkLoginHandler.java

```diff
@@ -83,40 +90,32 @@ public void setHandleErrorName(final String handleErrorName) {
             this.handleErrorName =handleErrorName;
         }
 
-        public boolean getInstitutionLogin() {
```

Comment:
`institutionLogin` is a `boolean` field, of which the getter method should be `isInstitutionLogin()`. Before this change, both `isInstitutionLogin()` and `getInstitutionLogin()` present and do the same thing.